.travis.yml

```diff
@@ -9,54 +9,38 @@ dist: trusty
 
 env:
   global:
-    - EMULATOR_API=24
+    - EMULATOR_API=16
     - ANDROID_API=28
     - ANDROID_BUILD_TOOLS=28.0.3
     - ADB_INSTALL_TIMEOUT=20
     - ANDROID_TRAVIS=1
     - secure: iXVs65+lPjyTr60MsX9K2WpjJ5ijt4cnBE9PT6ouHSjkZjugotaNoX3nXZev8fKFLsH0JPxpsxBVy66jz6iOddMfLdzSiL2oOeeYNxyNThOdGp9R/JmYtivgrMm6v3e1zzCLvj9onOwBUVdWVLD/BXgYi7wP96rJFENlVjuszys=
     - secure: BXwZhWDFFXzdaTAPAvMKeOfnLHxkmdXoeMZ/GtVYkT62JxeuiaTW6Z6rT0gZKAEyxmUBxFX1CP+1m1Gi0cWrQHfTBbFDJO7iKu7mG9LcZhQ8Cm657yt2TBc+fnVqbG5OxXWhSBs1lF1gZJ6+ehEIJn3YjP6fDYv0fNnTGpYHCYo=
+    - secure: G4ylgymVEE7+/bPwWvI7fT56tiBEtDqP9UYkdrvpNdzNAJq2/A60MIgh9c25JtJVIbDfZccZeJfX8eCviid4mg1hwNiVUmpODbwuLCjTEEuEmL4T7JXJKfhyc7jRcwO4i+iMf2lJyxhuxrYUXLnoe1Ih4XKGGk8RNZtO22ko0qQ=
 
 android:
   components:
-    - tools
-    - platform-tools
-    - tools
     - build-tools-$ANDROID_BUILD_TOOLS
-    - android-$ANDROID_API
-    - android-$EMULATOR_API
-    - extra-android-support
-    - extra-google-m2repository
-    - extra-google-google_play_services
-    - extra-android-m2repository
-    - addon-google_apis-google-$ANDROID_API # google play services
-    - sys-img-armeabi-v7a-addon-google_apis-google-$ANDROID_API
-    - sys-img-armeabi-v7a-android-$EMULATOR_API # the fix!
 
 licenses:
   - android-sdk-license-.+
   - '.+'
 
-before_install: echo 'count=0' > /home/travis/.android/repositories.cfg
+before_install:
+  - echo 'count=0' > /home/travis/.android/repositories.cfg
 
 install:
-  - sdkmanager --list --verbose || true
   - yes | sdkmanager "tools"
-  - yes | sdkmanager "platform-tools"
-  - yes | sdkmanager "build-tools;"$ANDROID_BUILD_TOOLS
-  - yes | sdkmanager "platforms;android-"$ANDROID_API
-  - yes | sdkmanager "emulator"
-  - yes | sdkmanager "extras;android;m2repository"
-  - yes | sdkmanager "extras;google;m2repository"
-  - yes | sdkmanager --update
-  - yes | sdkmanager --licenses
-  - sdkmanager --list --verbose || true
+#  - docker pull snowplow/snowplow-micro:0.1.0
+#  - wget https://bin.equinox.io/c/4VmDzA7iaHb/ngrok-stable-linux-amd64.zip
+#  - unzip ./ngrok-stable-linux-amd64.zip
```

Comment:
If these are commented out do we need them?

Reply:
Micro integration is half complete, i.e. integration tests are run locally only, not in Travis, and I opened #328 to address the other half. These lines are commented out because they aren't necessary until 328 is done. Would you prefer to save this incomplete work somewhere else and delete from here? Or anything else?

Reply:
Maybe add a comment that these commented lines are related to that ticket and leave them in then?